src/Surfnet/StepupMiddleware/ApiBundle/Controller/RaCandidateController.php

```diff
@@ -76,16 +85,24 @@ public function searchAction(Institution $actorInstitution, Request $request)
 
     /**
      * @param Request $request
+     * @param Institution $actorInstitution
```

Comment:
Did you want to add this parameter to the `getAction` method? Or forgot to remove it at a later stage?

Reply:
The latter. Removed it.